README.md

```diff
@@ -57,5 +57,28 @@ From an LmP device:
 ~~~
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+For scenarios where you'd like to test the API call without involving a physical device or emulation:
+
+This project includes a simple [fake-lmp-device-register](./fake-lmp-device-register) within this project. 
+This tool mimics the behavior of `lmp-device-register`, enabling you to quickly test this project's API.
+
+## Running the fake-lmp-device-register
+
+### Dependencies:
+- Python 3
+- requests library
+- openssl command-line tool
```

Comment:
I'd defer this to @doanac but I think that that fake-lmp-device-register also has a dependency on CURL to test the full flow:
https://github.com/foundriesio/factory-registration-ref/blob/main/fake-lmp-device-register#L100

Reply:
I added curl as well